monai/transforms/utils.py

```diff
@@ -150,12 +150,13 @@ def zero_margins(img: np.ndarray, margin: int) -> bool:
 
 def rescale_array(
     arr: NdarrayOrTensor,
-    minv: float = 0.0,
-    maxv: float = 1.0,
+    minv: Optional[float] = 0.0,
```

Comment:
As you added support for `None`, could you please also change below line:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/intensity/array.py#L436
To:
`if self.minv is not None or self.maxv is not None:`

Thanks.

Reply:
sure, I updated the if condition.